_includes/openseeds-navbar.html

```diff
@@ -0,0 +1,77 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+    <div class="container">
+      <div class="navbar-brand">
+        <a class="navbar-item" href="{% link index.md %}">
+            <img src="/images/logo.png" alt="OLS">
```

Comment:
The logo should the one for Open Seeds

Reply:
Yeah, I had that thought also. The thing is, the openseeds logo is not in the _images_ directory...because we intend to add it from the theme.

Reply:
The OLS logo should be also in the theme also. No?

Reply:
Yes, it is in the theme.

But we haven't implemented the theme yet. So, I'm concerned that putting it in this particular PR might break things.

Reply:
Ah ok